app/views/announcements/index.html.erb

```diff
@@ -5,7 +5,7 @@
     </h4>
   </div>
   <div class="col-3">
-    <%= paginate @announcements, window: 2 %>
+    <%== pagy_bootstrap_nav(@pagy) if @pagy.pages > 1 %>
```

Comment:
With regards to the `if @pagy.pages > 1`, I've opened [this discussions](https://github.com/ddnexus/pagy/discussions/790).

Reply:
I created a helper which overrides the default `pagy_bootstrap_nav`.